pydicom/tests/test_fileset.py

```diff
@@ -38,7 +38,7 @@
 
 
 TEST_FILE = get_testdata_file('DICOMDIR')
-TINY_ALPHA_FILESET = get_testdata_file("tiny_alpha/DICOMDIR")
+TINY_ALPHA_FILESET = get_testdata_file("TINY_ALPHA/DICOMDIR")
```

Comment:
Changed these to avoid warnings - the names are save in CS values, which can only be upper-case.